guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -390,6 +390,38 @@ $shoppingCartUrl = $block->getData('shopping_cart_url');
 $helperMethodResult = $block->getData('helper_method_result');
 ```
 
+## Common arguments for blocks
+
+The following are common arguments for block instructions:
+
+- `cache_key`: key for saving/retrieving cached information. This is helpful if the block needs to be cached: [example]({{ page.baseurl }}/cloud/project/project-routes-more-cache.html).
+
+- `template`: sets the template for the block.
+
+    ```xml
+    <referenceBlock name="page.main.title">
+     <arguments>
+       <argument name="template" xsi:type="string">%Namespace_Module::new_template.phtml%</argument>
+     </arguments>
+   </referenceBlock>
+   ```
+
+- `translate_inline`: `true' = disable translation for this block.
```

Comment:
Seems like 'true' would enable translation, not disable it. Please confirm.

Reply:
Hi @dobooth, Yes, when it is `true` it would enable translation.